Update ask question message (#3814)
RasaHQ/rasa: .github/ISSUE_TEMPLATE/ask_question.md

---

<!--

> PLEASE DO NOT OPEN THIS ISSUE, IT WILL BE CLOSED:

Reviewer: ```suggestion
PLEASE DO NOT CREATE ISSUES TO ASK QUESTIONS, THEY WILL BE CLOSED:
```
